activestorage/app/models/active_storage/blob.rb

```diff
@@ -230,6 +234,29 @@ def purge_later
     ActiveStorage::PurgeJob.perform_later(self)
   end
 
+  # Returns an instance of service, which can be configured globally or per attachment
+  def service
+    custom_service =
+      if service_name
+        begin
+          ActiveStorage::ServiceRegistry.fetch(service_name)
+        rescue => e
+          msg = "Cannot configure service :#{service_name} for #{inspect}:"
+
+          case ActiveStorage.service_misconfiguration_behavior
```

Comment:
Let's imagine a following situation:

There are two services configured in the `yaml` - `:s3_public` and `:s3_private`, the first one is configured as a global default. App records some blobs with a service `:s3_private` and one day someone removes it from the config. In this case we have some blobs saved to the storage, which is not available anymore.

I think it would be helpful to notify a developer that the app is trying to access such a "broken" blob.